docs/UserGuide.md

```diff
@@ -870,21 +881,22 @@ Examples:
 
 ### Clearing all entries
 
-Clears all entries from VirusTracker.
+If you want to clear all entries from VirusTracker, you can use the following command.
 
 Format: `clear`
 
 ### Viewing help
 
-Shows a message explaining how to access the help page.
+If you are sure how to use VirusTracker and want get some help, you can use the following command which will show
```

Comment:
If you are unsure of how to use

Reply:
maybe can say how to access the user guide for the instructions?

Reply:
Fixed.